frontends/web/src/routes/settings/manage-accounts.tsx

```diff
@@ -264,46 +287,51 @@ class ManageAccounts extends Component<Props, State> {
             <View fullscreen={false}>
               <ViewContent>
                 <WithSettingsTabs deviceIDs={deviceIDs} hideMobileMenu hasAccounts={hasAccounts}>
-                  <>
-                    <Button
-                      className={style.addAccountBtn}
-                      primary
-                      onClick={() => route('/add-account', true)}>
-                      {t('addAccount.title')}
-                    </Button>
-
-                    {
-                      accountsByKeystore.map(keystore => (<React.Fragment key={keystore.keystore.rootFingerprint}>
+                  <Button
+                    className={style.addAccountBtn}
+                    primary
+                    onClick={() => route('/add-account', true)}>
+                    {t('addAccount.title')}
+                  </Button>
+                  {
+                    accountsByKeystore.map(keystore => (
+                      <React.Fragment key={keystore.keystore.rootFingerprint}>
                         <p>{keystore.keystore.name}</p>
                         <div className="box slim divide m-bottom-large">
-                          { this.renderAccounts(keystore.accounts) }
+                          {this.renderAccounts(keystore.accounts)}
                         </div>
-                      </React.Fragment>))
-                    }
-                    { accounts.length === 0 ? t('manageAccounts.noAccounts') : null }
-
+                      </React.Fragment>
+                    ))
+                  }
+                  {currentlyEditedAccount && (
```

Comment:
just one more question: if the Dialog is conditionally rendered it wont animate on small screen correct?